src/lir.rs

```diff
@@ -121,6 +121,7 @@ impl Translate for Expression {
         lir_translate_helper(self, &mut lir);
         lir.instructions.shrink_to_fit(); // shrink the memory
         #[cfg(debug_assertions)]
+        #[allow(clippy::all)]
```

Comment:
I think we should take care of the usage of `allow` and any other explicit or implicit check bypassing.
I'd suggest any use of `#[allow]` should come with brief comments to explain why this `allow` is needed.

Reply:
@ADD-SP @dndx I suggest appending this to the Houdini coding standard to keep a better code quality.

Reply:
resolved I think